index.bs

```diff
@@ -2094,6 +2146,43 @@ operation has no [=authenticator attachment modality=] selection option, so the
 credential|credentials=]. The [=client=] and user will then use whichever is available and convenient at the time.
 
 
+### Resident Key Requirement Enumeration (enum <dfn enum>ResidentKeyRequirement</dfn>) ### {#attachment}
+
+<xmp class="idl">
+    enum ResidentKeyRequirement {
+        "forbidden",
+        "indifferent",
+        "preferred",
+        "required"
+    };
+</xmp>
+
+This enumeration's values describe the [=[RP]=]'s requirements for [=resident credentials=]:
+
+<div dfn-type="enum-value" dfn-for="ResidentKeyRequirement">
+    :   <dfn>forbidden</dfn>
+    ::  This value indicates the authenticator MUST only create a non-resident credential.
```

Comment:
I'm not sure if you're asking me to take a stab at that or not? I'm happy to make an update to reference such a definition because I agree it's better than saying _MUST only create a non-resident credential_.

Reply:
I've taken a stab at it. I've made the shorthand alias _Non-Resident Credential_ and understand this may be debatable, however after consideration there is a LOT of existing references to the term _resident credential_, and I believe that the _Non-Resident Credential_ shorthand is the most logically definition of one that is **not client-side resident**.

Reply:
Ok, that makes sense.